install_config/configuring_authentication.adoc

```diff
@@ -290,6 +290,7 @@ oauthConfig:
     provider:
       apiVersion: v1
       kind: KeystonePasswordIdentityProvider
+      url: https://keystone_url:5000/v2.0
```

Comment:
I tested with v2.0 since that was the default on devstack, from what I can tell Gophercloud would match the url suffix to detect if it's v3 or v2 : 

https://github.com/openshift/origin/blob/0c674d492cf2cdc026a8549e12b49972299c116b/Godeps/_workspace/src/github.com/rackspace/gophercloud/openstack/client.go#L66-L69

I can surely specify v3 by default and note that v2 is supported as well,